popSimManu.tex

```diff
@@ -659,42 +665,43 @@ \section*{Methods}
 %\subsection*{Generating the PopSim resource}
 % RNG: No need to empty subsection.
 
-\subsection*{Model QC procedures}
+\subsection*{Model quality control}
 As a consortium we have agreed to a standardized procedure for model inclusion
 into \stdpopsim that allows for rigorous quality control. Imagine Developer A
 wants to introduce a new model into \stdpopsim. Developer A implements the
 demographic model for the relevant organism along with clear documentation
-of the model parameters and populations. This model is submitted as a Pull Request,
+of the model parameters and populations. This model is submitted as a ``pull request'',
 where it is evaluated by a reviewer and then included as `preliminary',
 but is not linked to the online documentation nor the command line interface.
-Inclusion of a preliminary model triggers a QC issue, and a second developer,
+Inclusion of a preliminary model triggers a quality control (QC) issue, and a second developer,
 Developer B, then independently reimplements the model from the relevant
 primary sources and adds an automatic unit test for equality between the
 QC implementation and the preliminary production model. If the two
-implementations are equal the original model is included in \stdpopsim.
+implementations are equivalent, the original model is included in \stdpopsim.
 If not, we move to an arbitration process whereby A and B first try
 to work out the details of what went wrong. If that fails, the original
 authors of the published model must be contacted
-to resolve ambiguities. This process can lead to cycles of revision of
-the implementation from Developers A and B but allows the project to assure
-quality. Indeed we have already had a number of interesting cases of arbitration
-to this point. Further details of our QC process can be found in our
+to resolve ambiguities. 
+% This process can lead to cycles of revision of the implementation from Developers A and B but allows the project to assure quality.
+% Indeed we have already had a number of interesting cases of arbitration to this point. 
+Further details of our QC process can be found in our
 \href{https://stdpopsim.readthedocs.io/en/latest/development.html#}{developer documentation}.
```

Comment:
This hyper ref is not displaying the link.
![Screen Shot 2019-12-20 at 1 15 17 PM](https://user-images.githubusercontent.com/26617086/71280993-c9d50180-232a-11ea-8364-5b50d632156d.png)

Reply:
whoops; meant to do `url`